cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/track-popover-control.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+
+import {
+    Row,
+    Col,
+    Select,
+    InputNumber,
+    Tooltip,
+} from 'antd';
+
+import Text from 'antd/lib/typography/Text';
+
+import { TrackerInputs } from 'reducers/interfaces';
+
+interface Props {
+    tracker_type: string | null;
+    tracker_until: string | null;
+    tracker_frame_number: number;
+
+    handleChange(name: string, value: string | number): void;
+}
+
+const trackTypeOptions = [
+    {
+        label: "OpenCV Boosting",
+        value: "Boosting",
+    },
+    {
+        label: "OpenCV MIL",
+        value: "MIL",
+    },
+    {
+        label: "OpenCV KCF",
+        value: "KCF",
+    },
+    {
+        label: "OpenCV CSRT",
+        value: "CSRT",
+    },
+    {
+        label: "OpenCV MedianFlow",
+        value: "MEDIANFLOW",
+    },
+    {
+        label: "OpenCV TLD",
+        value: "TLD",
+    },
+    {
+        label: "OpenCV MOSSE",
+        value: "MOSSE",
+    },
+    {
+        label: "GOTURN",
+        value: "GOTURN",
+    },
+]
```

Comment:
Let's avoid hardcoding of tracker types on client side. I would suggest two options here:
1. Get these types from the server
2. Implement UI only for one of tracker as the initial version

In the future we want to be able to work with different trackers, implemented as serverless functions.

Reply:
That server side function is not implemented yet but I think we should be able to add it easily as we move forward.

Reply:
@TheNaman047 

It should be implemented before we merge this PR, since our users will have questions about new controls in UI, that are useless for them right now, because our server doesn't support tracking yet.

Reply:
@bsekachev As per our discussion with Nikita, this PR was supposed to be for UI only since you guys are implementing a serverless function for tracking. We can update UI assuming there is an API, let's say `get_trackers_list`, and make appropriate changes in the front-end. I understand we can't merge it until we have the backend. I suggest we merge this PR and PR for the serverless tracking function together, once we have it.

Reply:
@savan77 Sure, this PR is for UI only. I mean UI shouldn't have hardcoded tracker types and display controls related with tracking by default (at this moment, till we haven't implemented server tracking). I think making temporary stubs (roughly speaking: ``const supportTracking = (): boolean =>  false`` and ``const getTrackers = (): any[] => [])`` is a suitable option). You do not need even make real server requests.

Reply:
Thanks @bsekachev for the inputs. @TheNaman047 can you make appropriate changes?

Reply:
@bsekachev For dynamic control display, I didn't get a chance to look at the `deep extreme cut` but what if we do something like this: fetch a list of available trackers from the backend (as we discussed above), if it returns at least one tracker then show that Tracker Setting button (or controls) or else don't show them. what do you think about that? In any case, I will ask @TheNaman047 to take a look at Deep Extreme Cut.

Reply:
@savan77

I think this option is also suitable. But, I suppose in this case if server raises 404 code, UI also shouldn't display any tracking controls. The key idea of PR is that UI should assume some data types and be ready work with them immediately after server is able to provide real API.

Reply:
Yes, that's right. We won't display Tracking Controls if there is any error. We will display it only and only if there is at least one tracker available. @TheNaman047 can you also take a look at this? Thanks!

Reply:
@savan77 @TheNaman047 
Hi guys, are there any updates on it?

Reply:
@bsekachev       @TheNaman047 is working on other priority. He will probably get back to this next week.

Reply:
@bsekachev can you check now? @TheNaman047 pushed changes which checks of at least one tracker to show the tracker settings.

Reply:
@savan77 Sure, I will. But as I can see, we still have hardcoded trackers on UI (from OpenCV). Is it outdated code?

Reply:
@bsekachev that's right. @TheNaman047 As we discussed, we should not show any trackers by default. We need to show tracker settings only if we get positive response from that API call. And show that trackers in the list, not the hard-coded once.
Please correct me if I am wrong. @bsekachev

Reply:
@savan77 

Yes, there were two ideas in this thread above. The first is what you mentioned and the second is do not hardcode trackers on client side. UI should get these types from the server. Right now one of trackers is implemented as a nuclio serverless function and we might use it in the implementation. 

I understand if it isn't your priority, on our side I am ready to modify the PR next week

Reply:
Thanks @bsekachev , that would be great since you have some ideas about this new serverless function.